src/types/shipping_address.rs

```diff
@@ -1,9 +1,23 @@
-#[derive(Debug, Deserialize, Hash, PartialEq, Eq, Clone, Serialize)]
+/// This object represents a shipping address.
+///
+/// [The official docs](https://core.telegram.org/bots/api#shippingaddress).
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
 pub struct ShippingAddress {
+    /// ISO 3166-1 alpha-2 country code.
     pub country_code: String,
```

Comment:
Should we add `// TODO checked ISO 3166-1 alpha-2 type`?

Reply:
And an Issue.